components/basecomponents/KidsTranslation.tsx

```diff
@@ -1,45 +1,86 @@
+import { useCallback } from 'react';
 import { useTranslation, Trans } from 'next-i18next';
-import React, { useCallback, useState } from 'react';
+
+/** Components */
 import PlayKidsIcon from '../../public/icons/play.svg';
-import { Language } from '../../utils/locales';
-import { AudioPlayer } from 'utils/AudioPlayer';
 import { Flag } from './Flag';
+import { AudioPlayer } from 'utils/AudioPlayer';
+
+/** Hooks, Types, Utils */
 import styles from './KidsTranslation.module.css';
-interface KidsTranslationProps {
-  translation: string;
-  transcription: string;
-  soundUrl: string;
-  language: Language;
-}
-
-export const KidsTranslation = ({ transcription, translation, language, soundUrl }: KidsTranslationProps): JSX.Element => {
+import { Platform, KidsTranslationProps } from '@types';
+import { useAtom, useAtomValue } from 'jotai';
```

Comment:
Is jotai really really helpful/necessary here? There is a complexity cost associated with adding another state management library to a project. It seems the atoms are mostly being used by components with parent-child relationships. Consider if you cannot achieve the same functionality with simple `useState` and props.

Reply:
Even for keeping track of the platform, you can just have a function that checks the URL, no need to manually manage this as state.

Reply:
If I will need to check platform in different components, I would need to create i.e. new custom hook and return current platform. With Jotai, it is much easier. 

For other atoms, there are multiple levels between components and for example for if is playing sound, I need to go with state up from botton. I used Jotai, because it is really easy to grab and use it. With React useContext, it will cause rerender of whole components tree for all components depends on the context.

With more games, I think it will be really easier to go with this state management.